src/Asm89/Twig/Lint/Command/LintCommand.php

```diff
@@ -64,6 +72,7 @@ protected function execute(InputInterface $input, CliOutputInterface $output)
         $twig     = new StubbedEnvironment(new \Twig_Loader_String());
         $template = null;
         $filename = $input->getArgument('filename');
+        $this->bOnlyPrintErrors = $input->getOption('only-print-errors');
```

Comment:
Make this a local variable (drop the `b` prefix) and pass it do the `validateTemplate()` method instead.